app/views/courses/edit.html.erb

```diff
@@ -1,12 +1,23 @@
 <h1>Edit course</h1>
 
+<%
+   custom_points_url_help = %q{
+     <small>
+       {user} = username<br/>
+       {course} = course id<br/>
+       {org} = organization id<br/>
+     </small>
+   }
+%>
```

Comment:
how this would look? (screenshot plz) (you know you can just drag and drop screenshots to the comment fields)